File: nested_loop/1.c
#include <stdio.h>

int main(){
  int i, j, sum = 0;

  for(j = 0; j < 100; j++) {
    for(i = 0; i < 1000000; i++){
      sum += i;
      sum = sum % 7123;
    }
  }
  /*   printf("%d\n", sum); */
}

File: sum/1.js
var sys = require("sys");
var start = (new Date);

var sum = 0, i;

for(i = 0; i < 1000000; i += 1) {
    sum += i;
}

var end = (new Date);
var run_time = end - start;

sys.puts("this took: " + run_time);
